.gitattributes

```diff
@@ -1 +1,2 @@
 khmer/_version.py export-subst
+ChangeLog    merge=merge-changelog
```

Comment:
Good to know. Something to investigate for another day.

On Sun, Jun 14, 2015 at 12:36 PM C. Titus Brown <notifications@github.com>
wrote:

> In .gitattributes
> <https://github.com/dib-lab/khmer/pull/1031#discussion_r32383396>:
>
> > @@ -1 +1,2 @@
> >  khmer/_version.py export-subst
> > +ChangeLog    merge=merge-changelog
>
> On Sun, Jun 14, 2015 at 12:34:20PM -0700, Michael R. Crusoe wrote: > @@ -1
> +1,2 @@ > khmer/_version.py export-subst > +ChangeLog merge=merge-changelog
> Thanks for catching that. Probably shouldn't be merged as not everyone will
> have git-merge-changelog installed. I've moved it to my personal site-wide
> configuration file.
> (It doesn't seem to be a problem for merging, note; I ran a merge with it
> set, and no problem.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1031/files#r32383396>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
<mcrusoe@msu.edu>
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe